tachyon/zk/plonk/circuit/phase.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020-2022 The Electric Coin Company
+// Copyright 2022 The Halo2 developers
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.halo2 and the LICENCE-APACHE.halo2
+// file.
+
+#ifndef TACHYON_ZK_PLONK_CIRCUIT_PHASE_H_
+#define TACHYON_ZK_PLONK_CIRCUIT_PHASE_H_
+
+#include <stdint.h>
+
+#include <string>
+
+#include "tachyon/base/strings/string_number_conversions.h"
+#include "tachyon/export.h"
+
+namespace tachyon::zk {
+
+// NOTE(luke): Rotation class is copyable, assignable, and occupy 8 bits per
```

Comment:
It would be better to use your GitHub ID for tracking.
```suggestion
// NOTE(lightscale-luke): Rotation class is copyable, assignable, and occupy 8 bits per
```